client/my-sites/plans-v2/utils.ts

```diff
@@ -416,3 +418,43 @@ export function isUpgradeable( slug: string ): boolean {
 export function getProductUpsell( slug: string ): string | null {
 	return UPSELL_PRODUCT_MATRIX[ slug ];
 }
+
+/**
+ * Append "Available Options: Real-time and Daily" to the product description,
+ * if product/plan offers 'realtime' and 'daily' options, AND product/plan is not already owned,
+ *
+ * @param product SelectorProduct
+ * @param ownedProducts array | string
+ *
+ * @returns ReactNode | TranslateResult
+ */
+export const getJetpackDescriptionWithOptions = (
+	product: SelectorProduct,
+	ownedProducts: string[] | string | null
+): React.ReactNode | TranslateResult => {
+	if ( ! ownedProducts ) {
+		return product.description;
+	}
+
+	const productsOwned =
+		'item-type-plan' === product.type && 'string' === typeof ownedProducts
+			? [ ownedProducts ] // if it's a plan, make it an array
+			: ownedProducts; // otherwise, it's already an array
+
+	const em = React.createElement( 'em', null, null );
+
+	// check if 'subtypes' property contains daily and real-time product options.
+	// and check that this product is not owned.
+	return product.subtypes.some( ( subtype ) => DAILY_PRODUCTS.includes( subtype ) ) &&
+		product.subtypes.some( ( subtype ) => REALTIME_PRODUCTS.includes( subtype ) ) &&
+		! productsOwned.includes( product.productSlug )
```

Comment:
We can simplify this a bit knowing that a product that has subtypes (daily and real-time) won't ever be owned by a user because it's impossible to purchase them (you always have to choose from daily or real-time, you can't purchase the "generic/option" version). So, in the end, we can safely remove the last condition: `! productsOwned.includes( product.productSlug )`.

To give more context, the cards that have subtypes (daily and real-time) are "fake" products that only exist in the front-end. For instance, the Jetpack Security generic/option card has a product slug equal to `jetpack_security` and the real Jetpack Security Daily/Real-time products have as slug `jetpack_security_daily` and `jetpack_security_realtime` respectively. `jetpack_security` doesn't exist in the back-end, so it can be purchased/owned. The same happens with the generic/option card for Jetpack Backup.

Reply:
Ahh yes, I see.  Ok yes, that definitely simplifies it. 👍